CHANGELOG.md

```diff
@@ -17,3 +17,6 @@ This project adheres to [Semantic Versioning](http://semver.org/) and http://kee
 - Remove string slice and use a datetime function instead.
 - Now compatible with python 3.5.2
 - Added Contributor Dark-Passenger to Readme
+- 
+### Added
```

Comment:
Can you add the version?  I also think it's tracked in code.  Safe to assume it can be increased according to semantic versioning since you're the only current contributor.